R/ci.R

```diff
@@ -39,6 +39,15 @@ CI <- R6Class(
     is_tag = function() {
       stop("NYI")
     },
+    #'  \item{`is_env()`}{Is the given env variable set to the given value?}
+
+    is_env = function(env, value) {
+      Sys.getenv(env) == value
+    },
+    #'   \item{`has_env()`}{Does the given env variable exist?}
+    has_env = function(env) {
+      stop("NYI")
```

Comment:
Is this intended?

Reply:
I did not really know how to set the default correctly here as I do not really understand what you mean by "NYI" and if there is ever a situation in which this setting applies over the `ci_` subclasses? Feel free to change it :)